protocol/test-serialization/src/test_3NKaBJsN1SehD6iJwRwJSFmVzJg5DXSUQVgnMxtH4eer4aF5BrDK.rs

```diff
@@ -319,14 +302,20 @@ mod tests {
             }
         };
 
-        let hash_str = "jwHLk8kaC6B45K3sjuX2sM38649VtfpUAteTfKFQMPcqTeXjGiT";
-        let bytes = bs58::decode(hash_str).into_vec().unwrap();
-        assert_eq!(et.delta_transition_chain_proof.0.as_ref(), &bytes[2..34]);
-        // FIXME: Investigate why this fails
+        let bytes = bs58::decode("jwHLk8kaC6B45K3sjuX2sM38649VtfpUAteTfKFQMPcqTeXjGiT")
+            .into_vec()
+            .unwrap();
+        assert_eq!(et.delta_transition_chain_proof.0.as_ref()[..], bytes[2..34]);
+        // FIXME:
```

Comment:
next PR or missed?

Reply:
the hex representation from block json uses a different byte code here, which is unexpected, need to figure out why